extensions/reactive-datasource/runtime/src/main/java/io/quarkus/reactive/datasource/runtime/DataSourceReactiveRuntimeConfig.java

```diff
@@ -94,4 +95,22 @@
      */
     @ConfigItem
     public Optional<Boolean> threadLocal;
+
+    /**
+     * The number of reconnection attempts when a pooled connection cannot be established on first try.
+     */
+    @ConfigItem(defaultValue = "0")
+    public int reconnectAttempts;
+
+    /**
+     * The interval between reconnection attempts when a pooled connection cannot be established on first try.
+     */
+    @ConfigItem(defaultValue = "PT1S")
+    public Duration reconnectInterval;
+
+    /**
+     * The maximum time without data written to or read from a connection before it is removed from the pool.
+     */
+    @ConfigItem(defaultValue = "PT0S")
```

Comment:
Hmmm, it looks like this one should more be `Optional`?

Reply:
Done